app.json

```diff
@@ -4,6 +4,9 @@
         "ALLOWED_HOSTS": {
             "value": ".herokuapp.com"
         },
+        "BUILDPACK_RUN": {
+            "value": "pipenv run cookiecutter . --config-file cookiecutter/react_template.yaml --no-input -f"
+        },
```

Comment:
Default for `BUILDPACK_RUN` is react.
As a reminder, If you look lower in this file you'll see the `https://github.com/weibeld/heroku-buildpack-run.git` buildpack. Heroku buildpacks let you perform different tasks during the build. This one lets us run an arbitrary command. In this case it's cookiecutter.